plugins/module_utils/network/vyos/argspec/bgp_address_family/bgp_address_family.py

```diff
@@ -34,56 +31,95 @@
 class Bgp_address_familyArgs(object):  # pylint: disable=R0903
     """The arg spec for the vyos_bgp_address_family module"""
 
-    def __init__(self, **kwargs):
-        pass
-
     argument_spec = {
-        "running_config": {},
-        "state": {
-            "default": "merged",
-            "type": "str",
-            "choices": [
-                "merged",
-                "replaced",
-                "deleted",
-                "gathered",
-                "parsed",
-                "rendered",
-                "purged",
-                "overridden",
-            ],
-        },
         "config": {
             "type": "dict",
             "options": {
-                "neighbors": {
+                "as_number": {"type": "int"},
+                "address_family": {
+                    "type": "list",
                     "elements": "dict",
+                    "options": {
+                        "afi": {"type": "str", "choices": ["ipv4", "ipv6"]},
+                        "aggregate_address": {
+                            "type": "list",
+                            "elements": "dict",
+                            "options": {
+                                "prefix": {"type": "str"},
+                                "as_set": {"type": "bool"},
+                                "summary_only": {"type": "bool"},
+                            },
+                        },
+                        "networks": {
+                            "type": "list",
+                            "elements": "dict",
+                            "options": {
+                                "prefix": {"type": "str"},
+                                "path_limit": {"type": "int"},
```

Comment:
My experience was the path_limit was deprecated some time ago - I did not see it in 1.3 nor in newer version. I wanted to re-confirm and remove this option in my work on T6829

Reply:
I'd like to remove it with T6829, to keep the documentation and argspecs in sync with the then-current code. This is intended to bring things to the state of the code at the time of the PR (which is part of why I want to get it in ASAP).